apps/app/src/views/routes/Settings/components/users/UserItem.tsx

```diff
@@ -42,6 +44,9 @@ export const UserItem = ({
       </Td>
       <Td>{user.email}</Td>
       <Td textOverflow={'ellipsis'}>{roles}</Td>
+      <Td>
+        <UserItemActions userId={user.id}></UserItemActions>
+      </Td>
```

Comment:
This should only show up if you have proper permissions.

Reply:
done